shyamraghu/shyamportfolio.github.io
# Done the Enhancement

**What the PR Do?**
The PR is add some more languages on the animation on the home page 

Fixed #1 

![image](https://github.com/shyamraghu/shyamportfolio.github.io/assets/94217498/90a0bff1-3855-4f86-83eb-89502bc50d2b)


**Type of change** 
Enhancement

**How should this be tested**
-- Go to the home page 
-- After the languages animation
-- You can see the last is **python** while before the lastone is **c++**
